ui/src/message_popup/experience_page.jsx

```diff
@@ -159,16 +172,19 @@ export default React.createClass({
 
   renderInstructions() {
     const {scaffolding} = this.state;
+    const {query} = this.props;
+
     return (
       <VelocityTransitionGroup enter={{animation: "callout.pulse", duration: 500}} leave={{animation: "slideUp"}} runOnMount={true}>
-        <div>
+        <div className="instructions">
           <InstructionsCard 
-           itemsToShow={this.props.query}
+           itemsToShow={query}
            />
           <ScaffoldingCard
             initialEmail={this.context.auth.userProfile.email}
             scaffolding={scaffolding}
-            itemsToShow={this.props.query}
+            itemsToShow={query}
+            allowChoosingResponseMode={_.has(query, 'modes')}
```

Comment:
Could we handle this inside of `ScaffoldingCard` because `itemsToShow` already contains `query`?